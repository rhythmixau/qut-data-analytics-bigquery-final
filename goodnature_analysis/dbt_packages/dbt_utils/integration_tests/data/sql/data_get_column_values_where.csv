version https://git-lfs.github.com/spec/v1
oid sha256:73567fb480097601599e0075a90f3b05abdaa6090e115d12f4cb96efaa3da3ea
size 97
