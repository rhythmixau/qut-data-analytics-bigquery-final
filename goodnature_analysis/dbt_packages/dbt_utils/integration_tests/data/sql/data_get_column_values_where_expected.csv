version https://git-lfs.github.com/spec/v1
oid sha256:07060cea965a31500fdaff35718bfe14ec80686d0d9dafa4fbec498ac77ebecf
size 13
